tests/setup_cfg_fmt_test.py

```diff
@@ -296,6 +296,72 @@ def test_adds_long_description_with_readme(filename, content_type, tmpdir):
     )
 
 
+@pytest.mark.parametrize(
+    ('filename', 'content_type'),
+    (
+        ('README.rst', 'text/x-rst'),
+        ('README.markdown', 'text/markdown'),
+        ('README.md', 'text/markdown'),
+        ('README', 'text/plain'),
+        ('README.txt', 'text/plain'),
+        ('readme.txt', 'text/plain'),
+    ),
+)
+def test_readme_discover_does_not_prefer_adoc(filename, content_type, tmpdir):
+    tmpdir.join(filename).write('my project!')
+    other_files = [
+        'README.adoc',
+        'README.asciidoc',
+    ]
+    for other_file in other_files:
+        tmpdir.join(other_file).write('my project!')
```

Comment:
it's funny that you wrote a loop to DRY this and ended up with 6 lines of code instead of 2 -- DRY is dumb

Reply:
thanks for the reminder, got myself to read a lil post on it again. fixed